lua/lualine.lua

```diff
@@ -8,6 +8,10 @@ local theme_set = {}
 
 M.separator = '|'
 
+M.opt = {
+  icons_enabled = true,
```

Comment:
if we are going down the options route then all options should be placed in `opt` table (theme and separator as well)

Reply:
That's a good idea .

Reply:
Done